shell.js

```diff
@@ -162,6 +162,35 @@ module.exports = function() {
     }
   };
 
+  var profileProcess = function(args, system, cb) {
+    if (!_runner.isProcessRunning(args.process)) {
```

Comment:
should detect if this is a node process or not, if not, it should display an error to the user